### ISSUE TITLE
[Proposal] Лицехваты и рыгачка

### ISSUE THREAD

**Issue by Pimap3** [TauCetiStation/TauCetiClassic#10392]
#### Подробное описание проблемы
Если вкратце, то пропозал удалить рыгание от убийства лицехвата рядом с человеком, поскольку эта херня пробивает через биокостюмы, РИГи, газовые маски и шлемы.
Но самое смешное, это когда ты в толпе алиенов с кучей своих парней пытаешься отбиваться как на маринах, а потом рядом с вами умирает один лицехват и вы все вместе весело рыгаете, а затем вас все жрут из-за стана который нельзя избежать. Ещё смешнее, это помереть от асфиксии после рыгачки в противогаз

**Comment by kalazus** [comment]
Легче просто удалить эту ОП фишку, написанную на коленке. Хз зачем её добавляли изначально ведь лицехваты это не основной антаг, а работает как микро крик квиры без возможности защититься.

**Comment by Deahaka** [comment]
Лицехваты выполняют роль сапорта для убивания челов в наушниках с супер-ригом. До этого Симбака в соло не раз забегал с сотней пушек в кармане до гнезда и выносил квину без сопротивления. Сейчас так уже никто не может по вот этой причине.

**Comment by DarthSidiousPalpatine** [comment]
Не надо ничего менять, у алиенов 50% винрейт, они - идеальный режим.

**Comment by kalazus** [comment]
Скилл ишшуес.
Но вообще, у нас квина просто слишком ватная, медленная и с 1 полезной способностью. И 2-ое, ничего лицехваты против этого не сделают т.к. самоубиться об сбшника и подождать рвоту занимает больше времени чем требуется для убийства квины. Зато теперь всё сопротивление сб валится 1-им лицехватом, вбежавшим в толпу. Одно дело когда это делает квина, можно понять. Другое, когда гостроль.

**Comment by DarthSidiousPalpatine** [comment]
Нет, пусть их не трогают, режим-то забаланшен идеально :dovolen:

**Comment by Chip11-n** [comment]
Этот так называемый идеальный винрейт достигается либо выпилом гнезда раундстартом, либо победой ксенов. И только редкие раунды ломают это, когда либо мехи вписываются, либо спам турелей, либо экипаж проморгал сплавку всего чего только можно и 10+ минут спокойного развития гнезда ксенов

**Comment by Deahaka** [comment]
Ты так говоришь, как будто когда-либо хуманчики после обнаружения ксеносов решали их убить интеллектом, не давая расширится, застроив стенами всё и расставив оборону, постепенно задавливая до гнезда без потерь.
Такого не бывает, минимум 5 человек сразу же бегут суицидится в надежде чужика ранить или убить (а если у ксенов тотальное скилл ишью, то может и убить всё гнездо). И их никто не останавливает, всем пофиг, а алиенам вроде и норм.

**Comment by Pimap3** [comment]
> Лицехваты выполняют роль сапорта для убивания челов в наушниках с супер-ригом. До этого Симбака в соло не раз забегал с сотней пушек в кармане до гнезда и выносил квину без сопротивления. Сейчас так уже никто не может по вот этой причине.

Чтоб ты понимал. Чтобы так забегать соло в гнездо нужны нереально мощно прокаченная удача или просто случай. Такие вот чуваки чаще всего гасятся двумя хантерами в паре, две быстрые цели, которые могут тебя легко повалить, еще и в узком помещении таких диванных рембо оч быстро гасят. Если гнездо,  а тем более квина слились от этого, то это скилл ишью
